requirements.txt

```diff
@@ -5,4 +5,5 @@ python-multipart>=0.0.6
 Requests==2.32.2
 tabulate>=0.9.0
 pyjwt>=2.8.0
-platform_api_client @ git+https://github.com/CentML/platform_api_python_client.git@main
+cryptography==42.0.8
+platform_api_client @ git+https://github.com/CentML/platform_api_python_client.git@client/generate-new-clients
```

Comment:
please change this to main branch

Reply:
done, yeah i have initially pointed it to that branch for local development